src/libsyntax/ast.rs

```diff
@@ -1253,11 +1253,11 @@ pub enum ExprKind {
 /// item qualified with this `Self` type.
 ///
 /// ```ignore (only-for-syntax-highlight)
-/// <Vec<T> as a::b::Trait>::AssociatedItem
+/// <Vec<T> as a::b::Trait>::AssocItem
```

Comment:
these aren't internal datastructures, please keep the original here

Reply:
Thanks for your review. It's fixed.